MANIFEST

```diff
@@ -32,6 +31,7 @@ LICENSE
 MANIFEST			This list of files
 META.json
 META.yml
+NOTICE
```

Comment:
The manifest file should be removed.
It is only needed for CPAN and should be auto-generated.